src/common.ts

```diff
@@ -136,6 +136,7 @@ export namespace CommonSymbols {
   export const native = "native";
   export const indexof = "indexof";
   export const valueof = "valueof";
+  export const ReturnType = "ReturnType";
```

Comment:
Wondering if the aliases can be global type aliases instead of additional builtins

```ts
export type ReturnType<T> = returnof<T>;
```

Reply:
I haven't gotten around to fixing the aliases yet. Will work on that today.

Reply:
Does this latest fix satisfy your request?

Reply:
No, what I'm suggesting is to skip implementing aliases like `ReturnType` as an actual builtin, and to make them type aliases provided by stdlib instead, where possible.

Reply:
Alright. All set.

We just need a list of aliases you would like to make. Am I missing any more?